src/components/TopBar.js

```diff
@@ -12,21 +12,33 @@ const topBarTextColor = 'white';
 export const styles = {
   title: {
     color: topBarTextColor,
+    display: 'flex',
   },
   icon: {
     color: topBarTextColor,
+    display:'flex',
   },
+  topBarSectionStyle: {
+    display: 'flex',
+    justifyContent: 'space-between',
+  },
+  topBarLeftSection: {
+    display: 'flex',
+    alignItems:'center',
+  }
 };
 
 function TopBar({ classes, menuAction }) {
   return (
     <AppBar position="static">
-      <Toolbar>
-        <IconButton aria-label="Navigation" onClick={menuAction}>
-          <MenuIcon className={classes.icon} />
-        </IconButton>
-
-        <Typography className={classes.title} variant="h6">Serif.nu</Typography>
+      <Toolbar className={classes.topBarSectionStyle}>
+        <div className={classes.topBarLeftSection}>
+          <IconButton aria-label="Navigation" onClick={menuAction}>
+            <MenuIcon className={classes.icon} />
+          </IconButton>
+          <Typography className={classes.title} variant="h5">Serif.nu</Typography>
+        </div>
```

Comment:
If we put an empty line between the div and the Typography below it can really help break up the code visually, helping to mentally distinguish what's on the left and what's on the right.